addons/mail/static/src/discuss/call/common/call_actions.js

```diff
@@ -10,6 +10,7 @@ callActionsRegistry
         condition: (component) => component.rtc,
         name: (component) => (component.rtc.selfSession.isMute ? _t("Unmute") : _t("Mute")),
         isActive: (component) => component.rtc.selfSession?.isMute,
+        isTracked: true,
```

Comment:
Ok for now based on current implementation of tracked state in call menu.

At some point this will be replaced by call status of user, not a tracking of action. There's plan to harmonize the discuss actions (thread / message / call) so this will be done eventually.